cmd/kubeadm/app/cmd/token.go

```diff
@@ -138,6 +138,7 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 		"config", cfgPath, "Path to kubeadm config file (WARNING: Usage of a configuration file is experimental)")
 	createCmd.Flags().BoolVar(&printJoinCommand,
 		"print-join-command", false, "Instead of printing only the token, print the full 'kubeadm join' flag needed to join the cluster using the token.")
+	// TODO: This is wrong, the added flag here should be named --ttl, not --token-ttl
```

Comment:
Why is this wrong? The TTL is for the token so `--token-ttl` makes sense to me. `--ttl` seems ambiguous

Reply:
This is for the command `kubeadm token create`, where `--ttl` makes sense I think.
Well, at this point it should be `--ttl` as that has been a thing since v1.6 and we shouldn't break the CLI